.github/workflows/java-build-for-release.yml

```diff
@@ -0,0 +1,92 @@
+name: Build Java Release
+on:
+  push:
+    tags:        
+      # if you change this pattern, make sure jobs.strip-tag still works
+      - 'release/java/v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  ci:
+    uses: ./.github/workflows/java-build.yml
+
+  strip-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.version.outputs.version }}
+    steps:
+      - name: process tag 
+        id: version
+        run: |
+          echo "version=$(echo ${{ github.ref_name }} | cut -c15-)" >> $GITHUB_OUTPUT
```

Comment:
Can you add a comment like `strip "release/java/v"`, I had to think for a sec about what the number 15 meant.

Reply:
Or even

```
TAG=${{ github.ref_name }}
echo "version=${TAG#"release/java/v"}" >> $GITHUB_OUTPUT
```

could be clearer IMO

Reply:
good call.